python/tests/api/writer/test_whylabs.py

```diff
@@ -76,3 +76,10 @@ def test_api_key_prefers_parameter_over_env_var(self, results, caplog):
         os.environ["WHYLABS_API_KEY"] = "0123456789.any"
         with pytest.raises(ValueError):
             results.writer("whylabs").option(org_id="org_id", api_key="api_key_123.foo").write(dataset_id="dataset_id")
+
+    def test_writer_accepts_dest_param(self, results, caplog):
+        self.responses = responses.RequestsMock()
+        self.responses.start()
+        self.responses.add(PUT, url="https://api.whylabsapp.com", body=results.view().to_pandas().to_json())
+        with pytest.raises(ValueError):
+            results.writer("whylabs").option(api_key="bad_key_format").write(dataset_id="dataset_id", dest="tmp")
```

Comment:
it prob raises the `ValueError` with the api_key, right? wouldnt it be best to have an `assert true` happening with `dest`? or maybe catch the ValueError log to make sure it's "dest unrelated"

Reply:
That is correct, the parameter mismatch that occurs without the change to the WhylabsWriter fails before entering the write method. This test expects an api key format `ValueError`. Without my whylabs writer changes this test causes a
 `TypeError` failing to call write()

I was looking for a cheap way to verify we enter the write() call, but don't actually make a network call to whylabs. We can follow up and fix the mocks a bit more to make sure this test is better focused.

Note that this change is being more permissive than allowing `dest`, any extra parameters are ignored.

Reply:
Added TODO for test follow up and simplified a bit.